docs/tutorial.dox

```diff
@@ -432,6 +432,11 @@ examples are included with the netCDF Fortran and C++ APIs.
 - \ref example_sfc_pres_temp
 - \ref example_pres_temp_4D
 
+Additionally, there is a example program demonstrating
+how to use the filter API. This is C only and only accessible
+when built with automake currently.
+- \ref filter_example
```

Comment:
Probably quite easy to add it to the CMake build.

Try adding it to this line in examples/C/CMakeLists.txt:

`SET(exam_C_tests simple_xy_wr simple_xy_rd sfc_pres_temp_wr sfc_pres_temp_rd pres_temp_4D_wr pres_temp_4D_rd)
`
Then add the name of the created test file(s) to this line:

`SET(CLEANFILES sfc_pres_temp.nc simple_xy.nc pres_temp_4D.nc simple_nc4.nc simple_xy_nc4.nc)
`

and you should be good to go.